.github/workflows/check_gpg_key_not_expired.yml

```diff
@@ -13,6 +13,41 @@ jobs:
       - name: Check out repository
         uses: actions/checkout@v4
 
+      - id: slack
+        uses: slackapi/slack-github-action@v1.27.0
+        with:
+          channel-id: ${{ env.SLACK_BOT_CHANNEL_ID }}
+          payload: |
+            {
+              "attachments": [
+                {
+                  "color": "#FFFF00",
+                  "blocks": [
+                    {
+                      "type": "section",
+                      "fields": [
+                        {
+                          "type": "mrkdwn",
+                          "text": "<${{ env.WORKFLOW_RUN_URL }}|${{ env.WORKFLOW_RUN_NAME }}>"
+                        },
+                        {
+                          "type": "mrkdwn",
+                          "text": "*Status:*\n`In Progress`"
+                        }
+                      ]
+                    }
+                  ]
+                }
+              ]
+            }
+        env:
+          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+          # C07R9GSGKEU == #mondoo-ops
+          SLACK_BOT_CHANNEL_ID: "C07R9GSGKEU"
+          WORKFLOW_RUN_URL: ${{ github.event.workflow_run.html_url }}
+          WORKFLOW_RUN_NAME: ${{ github.event.workflow_run.name }}
```

Comment:
Perhaps set these on the job level, so we don't have them twice?

Reply:
cleaned those vars up as they were actually not needed, and moved the remaining to the job level